### PR TITLE
AP_OSD: add primary airspeed item

### PR DESCRIPTION
**Pull request by vierfuffzig**
Adds the primary airspeed sensor data, as requested on RCG thread. Ping @sh83

### CODE REVIEW

File: libraries/AP_OSD/AP_OSD_Screen.cpp [hunk 1]
```diff
@@ -214,6 +214,10 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info[] = {
     // @Path: AP_OSD_Setting.cpp
     AP_SUBGROUPINFO(aspd2, "ASPD2", 41, AP_OSD_Screen, AP_OSD_Setting),
     
+    // @Group: ASPD1
+    // @Path: AP_OSD_Setting.cpp
+    AP_SUBGROUPINFO(aspd2, "ASPD1", 42, AP_OSD_Screen, AP_OSD_Setting),
```
Comment (sh83) [inline libraries/AP_OSD/AP_OSD_Screen.cpp]: Aspd1?

**Comment by sh83** [comment]
Discussion about this here https://www.rcgroups.com/forums/showthread.php?3102183-Most-Integrated-Ardupilot-Flight-Controller-%28ChiBios%29/page215

**Comment by vierfuffzig** [comment]
thanks @sh83

**Comment by vierfuffzig** [comment]
@sh83 added param description and squashed.

wrapping multi-instance panels into single functions is is something i don‘t think i can do without a build / test environment. will try and do that when back home from travelling in separate PRs for respective panels.

File: libraries/AP_OSD/AP_OSD_Screen.cpp [hunk 2]
```diff
@@ -214,6 +214,11 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info[] = {
     // @Path: AP_OSD_Setting.cpp
     AP_SUBGROUPINFO(aspd2, "ASPD2", 41, AP_OSD_Screen, AP_OSD_Setting),
     
+    // @Group: ASPD1
+    // @Path: AP_OSD_Setting.cpp
+    // @Description: Displays airspeed reported directly from primary airspeed sensor
+    AP_SUBGROUPINFO(aspd1, "ASPD1", 42, AP_OSD_Screen, AP_OSD_Setting),
```
Comment (Hwurzburg) [inline libraries/AP_OSD/AP_OSD_Screen.cpp]: Basti, could you update the item descriptor here , ala #11810? thanks

**Comment by vierfuffzig** [comment]
adopted params, rebased.
@Hwurzburg found a couple of typos in added param descriptions, will fix in separate PR

**Comment by tridge** [comment]
this PR mistakenly changes the ChibiOS submodule

**Comment by vierfuffzig** [comment]
@tridge sorry, got a new mobile github client, didn‘t realize submodules had diverged. should be fixed now.

**Comment by tridge** [comment]
looks good now, thanks!

**Comment by rmackay9** [comment]
This is outside of my area but looks like it's been approved so merging

**Pull request merged**